### ISSUE TITLE
(rc2) unable to convert a Python 'unicode' object to a C++ 'QString' instance

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#4732]
From the error reporter (issue 26093):

```
TypeError: invalid result from MyTorrentsContentModel.data(), unable to convert a Python 'unicode' object to a C++ 'QString' instance
```

Unfortunately, there is not much info, but it seems that the conversion from a Python string to a C++ string fails. The error occurred on Arch Linux.

**Comment by devos50** [comment]
@ichorid could you have a look at this one?

**Comment by ichorid** [comment]
@devos50 , I've looked into the thing yesterday. This is a rare issue specific to Arch Linux. I've tried to reproduce it on Ubuntu by feeding Qt different string types, and I could not trigger it.

This problem is specific to Arch Some. Other projects have this problem too when users update Qt libraries. According to reports, it seems to fix itself after another Qt upgrade. This means we don't need to act on it, because we really can't fix it and it is too narrow.
